tessellation/src/basic_shapes.rs

```diff
@@ -803,26 +774,31 @@ pub fn stroke_circle<Output>(center: Point, radius: f32, tolerance: f32, output:
     }
 
     let angle = (0.0, 2.0 * PI);
+    let starting_point = center + vec2(1.0, 0.0) * radius;
 
-    let arc_len = 0.5 * PI * radius;
+    let arc_len = 2.0 * PI * radius;
```

Comment:
This one, however, is indeed meant to use `2.0 * PI * radius` since the `stroke_border_radius` call below handles the entire circle so we have to compute the number of segments for the entire circle rather than a quarter of it like we do in the other places.